systems/chromebook_oak/readme.md

```diff
@@ -13,7 +13,7 @@
 
 - lenovo chromebook n23 - hana
 - acer chromebook r13 - elm
-- lenovo chromebook 300e (mt8173 version) - hana
+- lenovo chromebook 300e (mt8173 version) - hana (tested with kernel 6.1.0-stb-mt7+)
```

Comment:
i would prefer not to add kernel versions here - the tested section is more ment to distinguish the devices already at least tested once successfully from the ones which nobody every tested at all so far ... once they are working they usually remain working and if not its worth to open a github issue for that as it would be a regression

Reply:
Thanks for the feedback. I will address it.